recipe/meta.yaml

```diff
@@ -186,7 +191,8 @@ outputs:
         - apache-airflow-providers-pagerduty >=3.8.1
         - apache-airflow-providers-papermill >=3.8.2
         - apache-airflow-providers-pgvector >=1.4.0
-        - apache-airflow-providers-pinecone >=2.1.1
+        # currently not solvable when apache-airflow-providers-pinecone is included
+        # - apache-airflow-providers-pinecone >=2.1.1
```

Comment:
Unfortunate to have to comment out the pinecone provider and I don't know exactly the nature of the conflict but in local testing, this seemed to be the simplest way to get things working.